lib/svgo.js

```diff
@@ -20,7 +20,7 @@ var SVGO = module.exports = function(config) {
     this.config = CONFIG(config);
 };
 
-SVGO.prototype.optimize = function(svgstr) {
+SVGO.prototype.optimize = function(svgstr, info) {
```

Comment:
why you don't pass that `info` data in the config?

Reply:
Yes, a separate prefix config option is supported now (see https://github.com/svg/svgo/pull/700/files#diff-623dc50b643d62febf2af73ea1856b51R120).